perf/roles/trex-ndr/defaults/main.yml

```diff
@@ -1,8 +1,10 @@
 ---
 
-trex_dir: /tmp/trex-core
+trex_install_dir: /opt/trex
 output_file: /tmp/ndr-results.json
 profile: vlan
 vlan: 212
 expected_pps: 1
 outer_left_mac: de:ed:de:ed:00:01
+it_interval: 20
+first_it_time: 150
```

Comment:
these two lines should go in the next commit.

Reply:
Updated ...